inference-engine/ie_bridges/java/tests/IECoreTests.java

```diff
@@ -66,9 +71,11 @@ public void testLoadNetworDeviceConfig(){
         assertTrue(executableNetwork instanceof ExecutableNetwork);
     }
 
+    @Test
     public void testLoadNetworkWrongDevice(){
```

Comment:
Missing space before brace (for all tests in this file).

Reply:
Fixed